.github/workflows/enforce-issue-template.yml

```diff
@@ -0,0 +1,35 @@
+name: enforce issue template
+on:
+  issues:
+    types: ['opened']
+jobs:
+  enforce-issue-template:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: "Look for a matching issue template"
+      shell: bash
+      run: |
+
+        # Look for a matching issue template
+        echo "${{ github.event.issue.body }}" > actual
+        for expected in $(ls .github/ISSUE_TEMPLATE/*.md 2> /dev/null); do
+          echo -n "$(basename $expected)? "
+          if diff -rub <(grep '^#' $expected) <(grep '^#' actual) > /dev/null; then
+            echo "👍 💃"
+            exit 0
+          else
+            echo "👎"
+          fi
+        done
+
+        # Failed to find a match! Close the issue.
+        echo "${{ github.token }}" | gh auth login --with-token
+        cat << EOF > comment
+        {"body": "Sorry, friend. As far as this ol' bot can tell, your issue does not use one of this repo's available issue templates. Please try again using a template so that we have the best chance of understanding and addressing your issue. (And if I'm confused, please [let us know](https://github.com/getsentry/.github/issues/new?title=template+enforcer+is+confused&body=${{ github.event.issue.html_url }}). 😬)\n\n----\n\n[![Did you see the memo about this?](https://user-images.githubusercontent.com/134455/104515469-e04a9c80-55c0-11eb-8e15-ffe9c0b8dd7f.gif)](https://www.youtube.com/watch?v=Fy3rjQGc6lA)"}
+        EOF
+
+        gh api "https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.issue.number }}/comments" \
```

Comment:
Maybe add a comment to replace this when https://github.com/cli/cli/issues/517 is addressed?